hashgraph/guardian
# Merging Develop into Main for 2.22 release (#3292)

* template xsls



* fix name



* [skip ci] Add swagger.yaml

* [skip ci] Add swagger.yaml

* update template



* GITBOOK-86: change request with no subject merged in GitBook

* GITBOOK-87: change request with no subject merged in GitBook

* GITBOOK-77: prernaa.agarwal's Feb 1 changes

* fixes



* fix check files



* update irec files timestamps [skip ci]



* GITBOOK-89: corrected capiltalization

* fix request vc document block style



* GITBOOK-90: change request with no subject merged in GitBook

* Corrected command in Readme

* GITBOOK-91: change request with no subject merged in GitBook

* Update readme.md of iREC with new timestamps

* update



* updating tests for new changes

* update



* hotfix 2.21.1: fix update worker settings propagation (#3203)

* fix update worker settings



* fix



* update version




* [skip ci] Add swagger.yaml

* update settings




* fix fint




---------






* updating api tests

* prepare api tests




* prepare api tests




* cypress api tests added for actions

* test fix




* remove old nats job




* add yarn action




* add yarn action




* add yarn




* test




* add git




* add git




* test yarn




* test starting services




* test starting services




* test starting services




* test starting services script




* cypress api tests added for actions

* cypress api tests added for actions

* cypress api tests in CI test

* cypress api tests in CI test

* cypress api tests in CI test

* fix



* fix services start




* remove onpush run




* added automatic publishing policies script



* cypress api tests in CI test

* cypress api tests in CI test

* cypress api tests in CI test

* update analytics



* added config



* GITBOOK-92: Made some alignment changes

* added publish-policies script



* update secrets




* update secrets




* update dockerfile image versions




* update dockerfile image versions for web-proxy




* fix



* fix



* fix



* update



* fix evc



* fix



* fix



* fix



* added package



* update



* [skip ci] Add swagger.yaml

* Updated template environments to include 'filebase' IPFS provider hint

* Validation for filebase client for worker service



* IPFS client update to include filebase usage, with simple (but not scalable path) to add additional providers



* Mini test suite for testing and ensuring IPFS client usage for a given configuration

* fixes for artifacts, accounts and policy api tests

* Added notes for Windows

* create policy data migrator



* fix



* filebase



* filebase



* fix lint



* Fixed verifiable presentation



* remove comment



* update



* GITBOOK-93: made minor changes

* fix frontend build



* fix build



* fix frontend



* use local ipfs node



* fixes for contracts, profiles, schemas and etc.

* update urls

* update urls

* fix discontinue time tooltip



* update



* update



* GITBOOK-94: Updated Timestamps

* added fixtures, and update some urls

* fix



* fix



* Update readme.md of VM0041

* Update README.md of VM0017

* Update readme.md of VM0007

* Update readme.md of PWRM0001

* Update README.md of GS AR

* Update README.md of Verra ARR

* Update readMe.md of VM0042

* Update readme.md of remote workflowpolicy

* Update readMe.md of GHGP

* fixed urls

* fix clear btn in migrate data dialog



* fixes for schemas, tags and settings API tests

* GITBOOK-95: Added III.H and III.AV guide

* GITBOOK-96: Added II.J Demo Guide

* update



* [skip ci] Add swagger.yaml

* GITBOOK-97: corrected guides

* Update readme.md of AMS-II.J

* Update readme.md of AMS-III.AV

* Update readme.md of AMS-III.H

* Update readme.md

* Update readme.md

* Update readme.md of Methodology Library

* fix



* fix



* fix



* fix



* fix loaders



* change loaders order



* fix



* fix



* fix



* [skip ci] Add swagger.yaml

* fix



* fix



* fix



* fix



* [skip ci] Add swagger.yaml

* added tests for tokens, tcs, update for policies

* update



* [skip ci] Add swagger.yaml

* fix



* fixes failed tests

* fix



* fixes failed tests

* fix



* fix



* update



* [skip ci] Add swagger.yaml

* fix



* fix



* added junit reporter and small fixes

* fix urls

* fix



* [skip ci] Add swagger.yaml

* edit filepath to reports

* GITBOOK-98: bug fixes

* GITBOOK-99: bug fixes

* GITBOOK-101: Updated Roadmap

* fix Bls12381G2Key2020



* GITBOOK-102: BYO DID UI added

* fix lint



* fix clear properties



* fix



* fix



* fix test



* GITBOOK-103: Added calculation section

* GITBOOK-104: Added ACM0007 demo guide

* GITBOOK-105: Added AMS-II.J Demo Guide

* GITBOOK-106: AMS II.G Demo guide added

* GITBOOK-107: AMS-III.AV Demo guide added

* Update readme.md of AMS-III.AV

* Update readMe.md of AMS II.G

* Update readme.md of ACM0007

* Update readme.md of II.J

* GITBOOK-108: Added BYOD APIs

* GITBOOK-109: Updated some corrections

* GITBOOK-110: Added Authentication Process in Users section

* GITBOOK-111: Added Schema Tree Demo video link

* GITBOOK-112: ACM0007 changes

* GITBOOK-113: AMS-III.AR demo guide updated

* Update readme.md of AMS-III.AR guide

* GITBOOK-114: Updated AMS-III.AR guide

* Update readme.md of AMR-III.AR guide

* fix filebase



* fix filebase validation



* fix lint



* fix



* fix



* fix serialize schema



* GITBOOK-115: Updated Frameworks

* GITBOOK-116: Added Auto Suggestion images

* GITBOOK-117: Example data screens updated

* Update readme.md of AMS-III.AV

* Add files via upload

* GITBOOK-118: policy diff screens corrected

* fix destroy worker client



* tests fixes and enchancement

* fixes for tests and reporters

* test check fix

* small fixes for tokens tests

* GITBOOK-119: Added AMS-I.D guide

* Create readme.md of AMS-I.D

* add expected tokens count



* api tests scheduler



* fix pipeline and generation test data



* bump version



* [skip ci] Add swagger.yaml

---------

**Description**:


**Related issue(s)**:

Fixes #

**Notes for reviewer**:


**Checklist**
